js/utils/messageModal.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
```

Comment:
This probably isn't the most robust way to encapsulate the modal. I imagine it would work well as it's own view.

But, at least for now, I think it should work.